ttps/discovery/macos/get-running-processes/get-running-processes.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Gather macOS running processes.
+description: |
+  This TTP uses ps to gather information about running processes on the system.
+mitre:
+  tactics:
+    - T0007 Collection
+  techniques:
+    - T1057 Process Discovery
+  subtechniques:
```

Comment:
can just leave out `subtechniques` if there are none, no need for empty list placeholder - we are still tightening up the mitre spec for ttpforge but I think that's where it'll land :)

Reply:
Ok! While you're hammering that out consider explicitly declaring the tactic id. T0007 should be TA0007 which is immediately recognizable as a tactic rather than a technique, and is less confusing if you're glancing at the code.